src/Core/Checkout/Document/Event/DocumentOrderCriteriaEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Document\Event;
+
+use Shopware\Core\Checkout\Document\Renderer\DocumentRendererConfig;
+use Shopware\Core\Checkout\Document\Struct\DocumentGenerateOperation;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\Event\GenericEvent;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Contracts\EventDispatcher\Event;
+
+#[Package('checkout')]
+class DocumentOrderCriteriaEvent extends Event implements GenericEvent
+{
+    public const CREDIT_NOTE_ORDER_CRITERIA_EVENT = 'credit_note.document.criteria';
+    public const DELIVERY_ORDER_CRITERIA_EVENT = 'delivery.document.criteria';
+    public const INVOICE_ORDER_CRITERIA_EVENT = 'invoice.document.criteria';
+    public const STORNO_ORDER_CRITERIA_EVENT = 'storno.document.criteria';
+
+    /**
+     * @param DocumentGenerateOperation[] $operations
```

Comment:
With our latest change, this is now
```suggestion
     * @param array<string, DocumentGenerateOperation> $operations
```

Would you be so kind and change the annotation [here](https://github.com/shopware/shopware/blob/trunk/src/Core/Checkout/Document/Event/DocumentOrderEvent.php#L25) as well. This place has been forgotten